docs/can-guides/commitment/recipes/canvas-clock/3-circle.js

```diff
@@ -1,24 +1,33 @@
-can.Component.extend({
+import { Component } from "//unpkg.com/can@5/core.mjs";
+
+Component.extend({
   tag: "analog-clock",
-  view: '<canvas id="analog" width="255" height="255"></canvas>',
+  view: '<canvas this:to="canvasElement" id="analog" width="255" height="255"></canvas>',
```

Comment:
Let’s use template literals to contain strings with quotes:

```js
view: `<canvas this:to="canvasElement" id="analog" width="255" height="255"></canvas>`,
```

There’s a few places where this‘ll need to be changed.

Reply:
Done!